packages/dd-trace/src/dogstatsd.js

```diff
@@ -190,6 +190,14 @@ class CustomMetrics {
   constructor (config) {
     const clientConfig = DogStatsDClient.generateClientConfig(config)
     this.dogstatsd = new DogStatsDClient(clientConfig)
+
+    this._boundFlush = this.flush.bind(this)
```

Comment:
Why are `_boundFlush` and `_flushInterval` object level properties, and not just `const boundFlush` and `const flushInterval`?